Python/pystate.c

```diff
@@ -293,7 +295,9 @@ _PyInterpreterState_DeleteExceptMain()
         if (interp->id_mutex != NULL) {
             PyThread_free_lock(interp->id_mutex);
         }
-        PyMem_RawFree(interp);
+        prev_interp = interp;
```

Comment:
```suggestion
        PyInterpreterState *prev_interp = interp;
```

Please define variable where it is initialized, as possible.

Reply:
@methane and about the compiler, this variable will be created on the stack, ok, but for each time or just one time?

Reply:
One (or zero) time.